opentelemetry-sdk/tests/metrics/integration_test/test_console_exporter.py

```diff
@@ -73,6 +77,56 @@ def test_console_exporter(self):
         self.assertEqual(metrics["attributes"], {"a": "b"})
         self.assertEqual(metrics["value"], 1)
 
+    def test_exp_histogram_exporter(self):
```

Comment:
What do you think about finding another home for this test? This file appears to be for integration testing of the console exporter. Maybe the right place is `test_point.py` which has other to_json tests. (I think you'll want to make this test look more like those tests once you see them as well.)